routes/warehousesRoutes.js

```diff
@@ -1,31 +1,10 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const knex = require('../knexfile');
+const knex = require("../knexfile");
+const { createNewWarehouse } = require("../controllers/warehousesController");
+const { deleteWarehouse } = require("../controllers/warehousesController");
```

Comment:
These two lines are referring to the same file, so we can simplify the import by merging them into one line. This improves readability and reduces redundancy.

from 
```
const { createNewWarehouse } = require("../controllers/warehousesController");
const { deleteWarehouse } = require("../controllers/warehousesController");
```
to 

`const { deleteWarehouse ,createNewWarehouse  } = require("../controllers/warehousesController");`

Reply:
great point! thank you I've updated now